stripe/stripe.github.io
# Refactor to Next.js and add blog section

## Description
This PR fixes some semantic issues and refactors the website to Next.js so we can use Markdoc for future improvements. 
As an MVP, it also adds a "Blog posts" section that fetches the latest posts from dev.to, displays the 3 latest ones and redirects to dev.to to read more.

_The code needs a bigger refactor that will be handled in future PRs._

**Before**

<img width="1789" alt="Screen Shot 2022-09-08 at 12 35 59 PM" src="https://user-images.githubusercontent.com/100887638/189401523-76861f8f-196b-4365-93dc-07c9c0027469.png">

**After**

![Screen Shot 2022-09-09 at 10 04 39 AM](https://user-images.githubusercontent.com/100887638/189404618-5bf0cb6c-9d9c-48f5-9e67-31ad434da4cb.png)

## Test plan

As this website is hosted using GitHub pages, it needs an index.html file at the root of the project. This file is created after running `npm run build`. This command creates an `out` folder with the prod files in it. **If you make changes to this PR, remember to move the prod files outside of the `out` folder before pushing changes to this PR.**

If you want to launch the site locally, you need to:
* Fetch this PR
* Run `python -m http.server 3000` and check `http://localhost:3000` in your browser.

If you want to make changes, you can run `npm run dev` to check the site locally. 

**If you make changes to this PR and want to be sure about what it will look like once published, run the python command and what displays in the browser should be what will be published once the PR is merged.** 

Hopefully, we'll update this workflow if we can move away from GitHub pages.
